src/openrct2-ui/windows/Ride.cpp

```diff
@@ -4218,11 +4217,6 @@ static void WindowRideMaintenancePaint(rct_window* w, rct_drawpixelinfo* dpi)
 
 #pragma region Colour
 
-static uint32_t WindowRideGetColourButtonImage(int32_t colour)
```

Comment:
Is the plan to merge this before the ride window refactor or after. It will have a merge conflict.

Reply:
Before. I already rebased the Ride window PR once, I don’t mind doing it again if necessary.